policy/policy-authority_test.go

```diff
@@ -9,29 +9,30 @@ import (
 	"testing"
 
 	"github.com/letsencrypt/boulder/core"
+	blog "github.com/letsencrypt/boulder/log"
 )
 
 func TestWillingToIssue(t *testing.T) {
 	shouldBeSyntaxError := []string{
-		``,																							// Empty name
```

Comment:
Yes, quite. That file hadn't been hit with "go fmt" yet.

Note that while it may look like lines were deleted, they actually had 80+ whitespace characters removed.